lib/health_cards/attribute_filters.rb

```diff
@@ -10,18 +10,21 @@ def self.included(base)
     # Class level methods for HealthCard class specific settings
     module ClassMethods
       # Define allowed attributes for this HealthCard class
-      # @param klass [Class] Scopes the attributes to a spefic class. Must be a subclass of FHIR::Model
+      # @param type [Class] Scopes the attributes to a spefic class. Must be a subclass of FHIR::Model
       # @param attributes [Array] An array of string with the attribute names that will be passed through
       #  when data is minimized
-      def allow(type: FHIR::Model, attributes: [])
+      def allow(type: nil, attributes: [])
+        raise ArgumentError, 'Must specify a base type for allowable attributes' if type.nil?
+
         allowable[type] = attributes
       end
 
       # Define disallowed attributes for this HealthCard class
-      # @param klass [Class] Scopes the attributes to a spefic class. Must be a subclass of FHIR::Model
+      # @param type [Class] Scopes the attributes to a spefic class. If not used will default to all FHIR resources.
+      # To apply a rule to all FHIR types (resources and types), use FHIR::Model as the type
       # @param attributes [Array] An array of string with the attribute names that will be passed through
       #  when data is minimized
-      def disallow(type: FHIR::Model, attributes: [])
+      def disallow(type: nil, attributes: [])
```

Comment:
So, leaving this blank applies it to all FHIR resources, and using `FHIR::Model` applies it to resources and elements? It's weird that `nil` is the value for that, but I guess it would also be weird if it were a magic string or something rather than a class.

What do you think about defining a constant in this class and using that as the default value?